image/templates/helm/stackrox-secured-cluster/templates/sensor-scc.yaml

```diff
@@ -49,6 +49,7 @@ volumes:
 
 {{- end }}
 
+# Despite the lack of 'anyuid' in this role, the name is kept as 'use-anyuid-scc' to avoid manual upgrade steps
```

Comment:
What is the long-term plan for this role? If it's with us to stay for good, then I think leaving the `anyuid` in the name might be a wrong trade-off.

Otherwise, if we plan getting rid of it at some point (sorry, I lost track of the discussion), then I think the name is confusing enough that it makes sense to put this comment in a `kubernetes.io/description` annotation so that there is at least a chance that it will be visible to users.

Reply:
If we make sensor & sensor-upgrader run under `restricted` SCC, then this role will not be needed. If we need more permissive SCC, then we need to keep it.

Reply:
Let's wait for @misberner 's comments regarding required manual upgrade steps, cause if we need to do something special for the upgrade to work, then we can rename it already now.